desc/geometry/surface.py

```diff
@@ -158,6 +158,8 @@ def Z_basis(self):
     @property
     def rho(self):
         """float: Flux surface label."""
+        if not (hasattr(self, "_rho")) or self._rho is None:
+            self._rho = 1.0
```

Comment:
Any reason you didn't do `return self.__dict__.setdefault("_rho", 1)` like we do for all the other classes?

Reply:
No reason.